templates/example_cell_clustering.ipynb

```diff
@@ -17,7 +17,9 @@
   {
```

Comment:
<p>Typo, "data scored in cell_som_cluster_avgs_name" should be "data stored in cell_som_cluster_avgs_name". Again, I think it'd be helpful to mention that "cap" is used for the z-score values, so change "all values are capped at the value specified in the cap argument prior to training" to "z-scored values are caped at the value specified in the cap argument prior to training."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>